### DISCUSSION TITLE
Fresh install generates yarn warnings

### DISCUSSION THREAD

**Discussion by adamcooper1386** [vercel/next.js#25096] [Help]
### What version of Next.js are you using?

current

### What version of Node.js are you using?

current

### What browser are you using?

n/a

### What operating system are you using?

mac

### How are you deploying your application?

na

### Describe the Bug

A fresh install using create-next-app using yarn generates unmet peer dependency warnings. 
<img width="756" alt="Screen Shot 2021-05-12 at 8 17 18 PM" src="https://user-images.githubusercontent.com/38062975/118072697-3eb13d00-b35f-11eb-880b-1a3cba6e1c10.png">

resolving the unmet peer dependency warnings simply generates more warnings, as the packages appear to depend on depreciated libraries. This is a problem as right from the start I can't get a clean install without warnings.
<img width="985" alt="Screen Shot 2021-05-12 at 8 19 59 PM" src="https://user-images.githubusercontent.com/38062975/118072902-a5cef180-b35f-11eb-81c2-3e2e44eaf74e.png">

### Expected Behavior

install without errors or warnings. Packages shouldn't depend on peer dependencies that aren't explicitly defined in the docs ( I looked at the getting started docs page and used exactly the command listed) or packages that are depreciated. I personally don't agree with maintainers marking working code as depreciated, but regardless, the package is depreciated.

### To Reproduce

you can figure that out. ```yarn create next-app``` will get you pretty far.

**Comment by timneutkens** [comment]
Just tried this but I don't get any of those warnings 🤔 Maybe you're on an old version of yarn?

**Reply by adamcooper1386** [reply]
Good call, 1.3.2

Not actually sure what the current version of yarn is, but 1.x seems like it isn't it.
